### PR TITLE
Add venue section

### PR DESCRIPTION
**Pull request by inesilva**
It adds the venue section, to be updated with the actual location of 2022

### CODE REVIEW

File: css/app.scss [hunk 1]
```diff
@@ -276,13 +278,99 @@ section {
...
+        margin-bottom: 1.5rem;
+      }
+
+      .map-block {
+        margin-top: 40px;
+      }
+
+      .content-text {
+        margin-bottom: 40px;
+      }
```
[the remark this answers is not in the document]
Comment (oscard0m) [reply css/app.scss]: I still see some space differences here.

**Review by oscard0m — APPROVED** [review]

**Review by marcoow — CHANGES_REQUESTED** [review]
Nice :+1:

Can we split this into separate PRs for the venue and the closing diner though? We'll announce the venue before the closing diner so need to merge separately.

Also, as a general practice, I'd recommend separating the formatting changes into their own PR as well – removing trailing spaces etc. is great but adding it all to this makes this PR unnecessarily big.

**Review by oscard0m — APPROVED** [review]

**Review by marcoow — COMMENTED** [review]
I think there's a bit of vertical space missing above the croissants here?

Bildschirmfoto 2022-07-13 um 11 09 00

**Comment by marcoow** [comment]
@jkintscher can you share in a comment here the real content for the location?

File: css/app.scss [hunk 2]
```diff
@@ -285,9 +285,104 @@ section {
...
+    }
+  }
+
   &.past-sponsors {
     margin-top: 100px;
+<<<<<<< HEAD
 
+=======
+>>>>>>> 053309e (Add venue section)
```
Comment (marcoow) [inline css/app.scss:385]: 👀 :)

**Review by oscard0m — APPROVED** [review]

**Review by marcoow — APPROVED** [review]

**Comment by jkintscher** [comment]
> @jkintscher can you share in a comment here the real content for the location?

Sure thing! We just received the address today, but I went stalking their website and this should have all the info as well as some photos that you can hopefully use: https://themragency.com/en/location/the-pavillon-rive-gauche/

If you need more details and/or pics, just let me know!

**Comment by marcoow** [comment]
@inesilva can you update this with the actual location data that Joschka linked above?

**Review by marcoow — APPROVED** [review]
Nice 👍 Please hold off merging now – we need to synchronize this with sending out an email etc.

**Review by balinterdi — APPROVED** [review]

**Pull request merged**